asilib/asi/rego.py

```diff
@@ -52,6 +54,10 @@ def rego(
         the ASI data time interval.
     alt: int
         The reference skymap altitude, in kilometers.
+    custom_alt: bool
```

Comment:
Reads well! I have a small edit request here (and other `custom_alt` kwarg description). This moves the geometry elaboration into a [note block](https://sublime-and-sphinx-guide.readthedocs.io/en/latest/notes_warnings.html).

"If True, asilib will calculate (lat, lon) skymaps assuming a spherical Earth. Otherwise, it will use the official skymaps (Courtesy of University of Calgary).

.. note::
     The spherical model of Earth's surface is less accurate than the oblate spheroid geometrical representation. Therefore, there will be a small difference between these and the official skymaps.
"